File: normal/03.js

// let oddNumber = 1

// console.log("1 dan 100 gacha bo'lgan sonlar ichida toq sonlar qatori")
// while (oddNumber <= 100){
//     if(oddNumber % 2 == 1){
//         console.log(oddNumber)
//     }
//     oddNumber = oddNumber + 1
// }


function oddNumber(number){
    console.log("1 dan 100 gacha bo'lgan sonlar ichidan faqat toqlari")
    while (number < 100){
        if(number % 2 == 1){
            console.log(number)
        }
        number++
    }
}

oddNumber(1)

File: normal/03.py
# oddNumber = 0

# print("1 dan 100 gacha bo'lgan sonlarning faqat toqlari qatori")
# while oddNumber < 100:
#     oddNumber = oddNumber + 1
#     if oddNumber % 2 == 1:
#         print(oddNumber)


def oddNumber():
    num  = int(input("Qaysi songacha toq sonlar chiqsin "))
    number = 0
    while number < num:
        number = number + 1
        if number % 2 == 1:
            print(number)


oddNumber()
